src/noit_metric_director.c

```diff
@@ -1027,8 +1027,14 @@ handle_metric_buffer(const char *payload, int payload_len,
         stats_add64(stats_msg_seen, 1);
         int rv = noit_message_decoder_parse_line(message, has_noit);
 
-        if(message->value.whence_ms) // ignore screwy messages
-          stats_set_hist_intscale(stats_msg_delay, mtev_now_ms() - message->value.whence_ms, -3, 1);
+        if(message->value.whence_ms) {
+          // ignore screwy messages
+          int64_t diff = mtev_now_ms() - message->value.whence_ms;
+          if(diff > 300000) {
+            mtevL(mtev_error, "[%zd] %.*s\n", diff, payload_len, copy);
```

Comment:
This doesn't look like a particularly actionable errorlog message. Is it more of a debug thing perhaps?

Reply:
this was debugging, not supposed to be part of this.